package.json

```diff
@@ -37,7 +37,8 @@
   },
   "homepage": "https://github.com/ipld/js-ipld-dag-cbor",
   "dependencies": {
-    "borc": "^2.1.0",
+    "borc": "hugomrdias/borc#fix/remove-node-globals",
```

Comment:
we should get a new borc release out soon, so this probably isn't ideal

Reply:
Yes ofc forgot to mention it in the op